- al_msg: arrived
  orientation:
    w: -0.00017889565812358735
    x: -0.795183806784293
    y: 0.6063654371306599
    z: -0.0019073745646599493
  position:
    x: 0.2909433382790395
    y: -0.054597308171628854
    z: 0.45791421322426523
- al_msg: arrived
  orientation:
    w: 0.0018637190338850317
    x: 0.7253597031078959
    y: -0.6875740331130401
    z: 0.0330420436301769
  position:
    x: 0.3818258695074174
    y: -0.05053480243502132
    z: 0.38377982707505565
- al_msg: targeting
  orientation:
    w: -0.001889594117664669
    x: -0.7254636996848395
    y: 0.6874595317628337
    z: -0.033139735395495286
  position:
    x: 0.381926121663905
    y: -0.050412547420682984
    z: 0.34397880436745176
- al_msg: targeting
  orientation:
    w: -0.0015672083581702442
    x: -0.7254141019630193
    y: 0.6874984328344665
    z: -0.033433955513868505
  position:
    x: 0.3821520277593052
    y: -0.050500729945121337
    z: 0.37337180727028696
- al_msg: arrived
  orientation:
    w: -0.0008065956099587451
    x: -0.7255366020326928
    y: 0.6873127044416151
    z: -0.03460108130580615
  position:
    x: 0.21360021243034383
    y: 0.05720369383709554
    z: 0.3953450584691627
